packages/empty-state/src/BasicEmptyState/BasicEmptyState.styles.ts

```diff
@@ -0,0 +1,38 @@
+import { css } from '@leafygreen-ui/emotion';
+import { Theme } from '@leafygreen-ui/lib';
+import { palette } from '@leafygreen-ui/palette';
+import { spacing } from '@leafygreen-ui/tokens';
+
+export const rootStyles = css`
+  padding: 40px;
+  display: flex;
+  gap: ${spacing[3]}px;
+  align-items: center;
+`;
+
+export const textContainerStyles = css`
+  max-width: 432px;
```

Comment:
It's what's specified on our designs and it's currently only used once in this file so I didn't see a need for a const.

Reply:
Brought this up with Rob - it's really just because it's what looks good haha. I could set this as a variable `const MAX_WIDTH = 432`, but it's only used once at the top of the file, so I don't see a need for it.